src/Backend/Modules/MediaLibrary/Actions/Index.php

```diff
@@ -147,9 +147,9 @@ private function getTotalFolderCount($dataGrids)
     private function parseFiles()
     {
         // add js
-        $this->header->addJS('jstree/jquery.tree.js', null, false);
-        $this->header->addJS('jstree/lib/jquery.cookie.js', null, false);
-        $this->header->addJS('jstree/plugins/jquery.tree.cookie.js', null, false);
+        $this->header->addJS('/src/Backend/Modules/Pages/Js/jstree/jquery.tree.js', null, false, true);
```

Comment:
Is it this library?
https://www.npmjs.com/package/jstree ?
if so, load it with yarn